src/NHibernate/Loader/Loader.cs

```diff
@@ -972,6 +972,29 @@ private void CheckVersion(int i, IEntityPersister persister, object id, object e
 					entry.LockMode = lockMode;
 				}
 			}
+
+			// #1226: If it is already loaded and can be loaded from an association with a property ref, make
```

Comment:
Can this please be extracted into an own method?

Reply:
Done.